mlflow/server/js/src/components/artifact-view-components/ShowArtifactHtmlView.js

```diff
@@ -83,7 +82,7 @@ class ShowArtifactHtmlView extends Component {
         this.setState({ html: event.target.result, loading: false });
       };
       fileReader.readAsText(blob);
-    });
+    }).catch(error => this.setState({ error: error, loading: false }));;
```

Comment:
This should also be added to the ShowArtifactTextView.js. Should I open a new pull request for this ?

Reply:
Yep, a separate PR for this in ShowArtifactTextView.js makes sense - ideally in that separate Pr, we could pull `fetchArtifacts` out into a utility so we don't need to copy the implementation across components :)